main.tf

```diff
@@ -174,7 +174,7 @@ resource "aws_route_table" "intra" {
 
   vpc_id = "${local.vpc_id}"
 
-  tags = "${merge(map("Name", "${var.name}-intra"), var.tags, var.intra_route_table_tags)}"
+  tags = "${merge(var.tags, var.intra_route_table_tags, map("Name", "${var.name}-intra-${var.intra_subnet_suffix}"))}"
```

Comment:
The order should be as before so that `Name` tag can be overridden.

Also, `intra` is duplicated in value.

Reply:
@antonbabenko - see the updated commit. I removed the duplicate intra. 
Not sure what you mean by "The order should be as before so that `Name` tag can be overridden." It's the same format as the previous fixes (see line 166, one of the mods introduced from #168 ), and includes the Name tag which will overwrite anyway.

Reply:
Line 177 should have `map(), tags, intra_subnet_tags` - in this order. This is for the reason that more specific tags can override upstream. (`intra_subnet_tags => tags => map()`)

See line 284 for the correct example.

Reply:
@antonbabenko - thanks for clarifying. Updated.